### ISSUE TITLE
Use it as an IDE

### ISSUE THREAD

**Issue by arqex** [Jam3/devtool#47]
Hi,

Thanks for the great tool.

I am thinking that it would be great to use the debugger as a code editor. I could add folders to the workspace right clicking the sources tab but I couldn't make it catch the debugger. And the folders are not saved through executions.

I know this may be a load of work, but it would be a nice feature to have.

Cheers,
Javi

**Comment by mattdesl** [comment]
Yes, see #27. We are blocked on Electron, but this feature would be pretty huge. Feel free to explore alternatives; it may be possible to shim the Node internals/requires to get certain things working.

**Comment by nelix** [comment]
Could this be done as an atom plugin?

**Comment by mattdesl** [comment]
Hmm not sure how that would work, but feel free to explore. :smile:

**Comment by darrrk** [comment]
You can use devtool as IDE.
Need add your project in Workspace , then map any local js file to identical remote file in **file://** section.
Refresh, and you can at the same time edit files and debug it. ;)
Even, you can use any other language with sourceMap support, for example LiveScript. And programming, as like in native javascript.
